apps/researcher/src/messages/nl/clerk.ts

```diff
@@ -0,0 +1,659 @@
+    },
+  },
+  signIn: {
+    start: {
+      title: 'Log in',
+      subtitle: 'om door te gaan naar {{applicationName}}',
+      actionText: 'Geen account?',
+      actionLink: 'Registreren',
+    },
+    password: {
+      title: 'Vul je wachtwoord in',
+      subtitle: 'om door te gaan naar {{applicationName}}',
+      actionLink: 'Gebruik een andere methode',
+    },
+    emailCode: {
+      title: 'Check je e-mail',
+      subtitle: 'om door te gaan naar {{applicationName}}',
+      formTitle: 'Verificatiecode',
+      formSubtitle:
+        'Voer de verificatiecode in die verzonden is naar je e-mailadres',
+      resendButton: 'Verstuur code opnieuw',
+    },
+    emailLink: {
+      title: 'Check je e-mail',
+      subtitle: 'om door te gaan naar {{applicationName}}',
+      formTitle: 'Verificatielink',
+      formSubtitle:
+        'Gebruik de verificatielink die verzonden is naar je e-mailadres',
+      resendButton: 'Verstuur link opnieuw',
+      unusedTab: {
+        title: 'Je kan deze tab sluiten.',
+      },
+      verified: {
+        title: 'Successvol ingelogd',
+        subtitle: 'Je zal weldra doorgestuurd worden',
+      },
+      verifiedSwitchTab: {
+        subtitle: 'Ga naar de oorspronkelijke tab om verder te gaan',
+        titleNewTab: 'Ingelogd in andere tab',
+        subtitleNewTab: 'Ga naar de pasgeopende tab om verder te gaan',
+      },
+      loading: {
+        title: 'Inloggen ...',
+        subtitle: 'Je zal weldra doorgestuurd worden',
+      },
+      failed: {
+        title: 'Deze verificatielink is niet geldig',
+        subtitle: 'Ga naar de oorspronkelijke tab om verder te gaan.',
+      },
+      expired: {
+        title: 'Deze verificatielink is verlopen',
+        subtitle: 'Ga naar de oorspronkelijke tab om verder te gaan.',
+      },
+    },
+    phoneCode: {...commonTexts.signIn.phoneCode},
+    phoneCodeMfa: {...commonTexts.signIn.phoneCode, subtitle: ''},
+    totpMfa: {
+      title: 'Tweestapsverificatie',
+      subtitle: '',
+      formTitle: 'Verificatiecode',
+      formSubtitle:
+        'Voer de verificatiecode in die is gegenereerd door je authenticator app',
+    },
+    backupCodeMfa: {
+      title: 'Voer een backupcode in',
+      subtitle: 'om door te gaan naar {{applicationName}}',
+      formTitle: '',
+      formSubtitle: '',
+    },
+    alternativeMethods: {
+      title: 'Gebruik een andere methode',
+      actionLink: 'Help',
+      blockButton__emailLink: 'Verzend link naar {{identifier}}',
+      blockButton__emailCode: 'Verzend code naar {{identifier}}',
+      blockButton__phoneCode: 'Verzend code naar {{identifier}}',
+      blockButton__password: 'Inloggen met je wachtwoord',
+      blockButton__totp: 'Gebruik je authenticator app',
+      blockButton__backupCode: 'Gebruik een backupcode',
+      getHelp: {
+        title: 'Help',
+        content:
+          'Als je geen toegang hebt neem dan contact op met de klantenservice en we helpen je verder.',
```

Comment:
We should probably revise these texts at some point - I don't think we have a 'klantenservice' :)